Forge/src/main/java/com/blamejared/crafttweaker/api/event/type/CTCustomBepRegistrationEvent.java

```diff
@@ -0,0 +1,43 @@
+package com.blamejared.crafttweaker.api.event.type;
+
+import com.blamejared.crafttweaker.api.zencode.scriptrun.IBepRegistrationHandler;
+import com.blamejared.crafttweaker.api.zencode.scriptrun.ICustomBepRegistrationEvent;
+import com.blamejared.crafttweaker.api.zencode.scriptrun.IScriptLoadingOptionsView;
+import net.minecraftforge.eventbus.api.Event;
+import org.openzen.zenscript.parser.BracketExpressionParser;
+
+/**
+ * Allows users to register custom implementations of {@link BracketExpressionParser}.
+ *
+ * <p>This event is invoked every time a new script run is started, even for syntax checks.</p>
```

Comment:
Speaking of, do we want that in the long run, or would we only do that for first-runs alter and cache all classes/beps/... that belong to a module later?